website/docs/d/vm_placement_policy.html.markdown

```diff
@@ -30,13 +39,41 @@ output "policyId" {
   value = data.vcd_vm_placement_policy.tf-policy-name.id
 }
 ```
+
+## Example Usage for tenant users
+
+Tenant users don't have access to Provider VDC information so the only way to retrieve VM Placement Policies is to
+fetch them using the VDC information. The only constraint is that the desired VM Placement Policy **must be assigned
+to the VDC**.
+
+```hcl
+data "vcd_org_vdc" "my-vdc" {
+  org  = "test"
+  name = "vdc-test"
+}
+
+data "vcd_vm_placement_policy" "tf-policy-name" {
+  name   = "my-policy"
+  vdc_id = data.vcd_org_vdc.my-vdc.id
+}
+
+output "policyId" {
+  value = data.vcd_vm_placement_policy.tf-policy-name.id
+}
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
 * `name` - (Required) The name VM Placement Policy.
-* `provider_vdc_id` - (Required) The ID of the [Provider VDC](/providers/vmware/vcd/latest/docs/data-sources/vcd_provider_vdc) to which the VM Placement Policy belongs.
+* `provider_vdc_id` - (Required for System admins) The ID of the [Provider VDC](/providers/vmware/vcd/latest/docs/data-sources/provider_vdc) to which the VM Placement Policy belongs.
+* `vdc_id` - (Required for tenant users; *v3.8.1+*) The ID of the [VDC](/providers/vmware/vcd/latest/docs/data-sources/org_vdc) to which the VM Placement Policy is assigned.
```

Comment:
This is a generic question/suggestion about "tenant" vs. "org" term across the whole PR docs. Should we use "org users" instead of "tenant users"?

E.g. here:
```suggestion
* `vdc_id` - (Required for Org users; *v3.8.1+*) The ID of the [VDC](/providers/vmware/vcd/latest/docs/data-sources/org_vdc) to which the VM Placement Policy is assigned.
```

The thought behind this is that "tenant" is more a business term, while inside the CD itself we see "organization".

Comments are welcome!
CC: @dataclouder @Didainius

Reply:
Either we use `provider/tenant` (as it is sometimes appropriate for wide operations guide) or we use `System administrator/Org user`. 
While I have found the `provider/tenant` pair clearer in some contexts, I think that in this case it should be `Org users`.